### ISSUE TITLE
Introduce keys command (this is great piece of software)

### ISSUE THREAD

**Issue by alija83** [sabledb-io/sabledb#19]
I dont really need the command, but it could be useful for someone.

`KEYS pattern`

https://redis.io/docs/latest/commands/keys/

Your software performs really well. Good job! :)

Other notes:
I like that: 
 * compression of data is enabled by default
 * does not generate large log files as in may other projects (that also rely on rocksdb)
 * very low RAM usage

![Screenshot from 2024-06-21 09-59-06](https://github.com/sabledb-io/sabledb/assets/6471608/62b81fda-6ea2-4016-b273-a2de1e47a3bc)

I have 16K keys loaded.

Here I am doing some heavy lookups:
![Screenshot from 2024-06-21 10-02-15](https://github.com/sabledb-io/sabledb/assets/6471608/40ab906e-ffe8-486f-8919-791db3763bfa)

The performance metrics for command HGETALL is between 100000 ns and 220000 ns, but mostly around 120000 ns in my Java program. 
In other words, we are talking about 0.1 and 0.22 milliseconds. With the aerospike db, I was able to get to 0.05 millisecond with the free open-source edition, so no data compression.
FYI, my hdd is a "nvme0n1  WD_BLACK SN850X 1000GB"

**Comment by sabledb-io** [comment]
Thanks for the feedback. If you have other commands that you are missing - please open a bug / issue per command and I will do my best to add them
